alibi/explainers/anchor_base.py

```diff
@@ -835,7 +836,7 @@ def __init__(self, samplers: List[Callable], **kwargs) -> None:
         self.pool = ActorPool(samplers)
         self.samplers = samplers
 
-    def _get_coverage_samples(self, coverage_samples: int, samplers: List[Callable] = None) -> np.ndarray:
+    def _get_coverage_samples(self, coverage_samples: int, samplers: List[Callable]) -> np.ndarray: # type: ignore
```

Comment:
We ignore the type here because this method is an override of the one in the base class, except here `samplers` really is required.